src/Pages/Auth.elm

```diff
@@ -1,64 +1,43 @@
-module Pages.Auth exposing (Model, Msg(..), TypeUrl(..), Url, authUrlsDecoder, getAuthUrls, init, showAuthUrl, subscriptions, toSession, update, urlDecoder, urlTypeDecoder, view)
+module Pages.Auth exposing (Model, Msg(..), init, subscriptions, toSession, update, view)
 
 import Asset exposing (..)
+import Browser.Navigation as Nav
 import Element exposing (..)
+import Element.Font exposing (..)
 import Endpoint
-import Http
-import Json.Decode as JD
 import Page
 import Route
 import Session exposing (..)
+import UI
 
 
 
 -- Model
 
 
 type alias Model =
-    { session : Session
-    , urls : List Url
-    }
-
-
-type alias Url =
-    { url : String
-    , typeUrl : TypeUrl
-    }
-
-
-type TypeUrl
-    = Google
-    | Github
+    Session
 
 
 init : Session -> ( Model, Cmd Msg )
 init session =
-    ( Model session [], getAuthUrls )
+    ( session, Nav.load <| Endpoint.toString Endpoint.authUrls )
```

Comment:
When the user access the auth page, Elm will redirect automatically to the authentication service page.